public/v4/apps/yagpdb.yml

```diff
@@ -0,0 +1,89 @@
+captainVersion: 4
+
+services:
+    # YAGPDB
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-redis
+            - $$cap_appname-postgres
+        environment:
+            YAGPDB_OWNER: $$cap_yagpdb_bot_owner
+            YAGPDB_CLIENTID: $$cap_yagpdb_bot_client_id
+            YAGPDB_CLIENTSECRET: $$cap_yagpdb_bot_client_secret
+            YAGPDB_BOTTOKEN: Bot $$cap_yagpdb_bot_token
+            YAGPDB_HOST: $$cap_appname.$$cap_root_domain
+            YAGPDB_PQHOST: srv-captain--$$cap_appname-postgres
+            YAGPDB_PQUSERNAME: yagpdb
+            YAGPDB_PQDB: yagpdb_prod
+            YAGPDB_PQPASSWORD: $$cap_postgres_password
+            YAGPDB_REDIS: srv-captain--$$cap_appname-redis:6379
+        volumes:
+            - $$cap_appname-soundboard:/app/soundboard
+        caproverExtra:
+            dockerfileLines:
+                - FROM teyker/yagpdb:$$cap_yagpdb_version
+                - CMD ["/app/yagpdb", "-all", "-pa", "-exthttps=true", "-https=false"]
+    # Redis
+    $$cap_appname-redis:
+        image: redis:alpine
```

Comment:
`alpine` is not a fixed version. Use tags like `6.2.5-alpine` to ensure the version is fixed.